### DISCUSSION TITLE
Stamping out priority bugs

### DISCUSSION THREAD

**Discussion by huumn** [stackernews/stacker.news#1007] [General]
Hey @stackernews/foss @stackernews/eng. Thank you all for being here!

I know we have a lot of PRs waiting for review. I'll get to them ASAP once I fix some of these priority bugs. Like this one: https://stacker.news/items/489749

It shouldn't take more than a day or two.

If I'm blocking something important that you need clarification on in the meantime, feel free to @-mention me.

**Comment by SatsAllDay** [comment]
No worries dude!

**Comment by benalleng** [comment]
Yeah, no sweat! Take your... time... I'll see myself out

**Comment by huumn** [comment]
@stackernews/foss @stackernews/eng I'll do payouts and remaining code reviews tomorrow morning.

**Comment by felipebueno** [comment]
Hi @huumn

![image](https://github.com/stackernews/stacker.news/assets/241271/2a20fafb-38a6-41a6-a83e-15c887987012)

I was taking a look at the awards.csv files and I noticed there are two payments due to 2024-04-24. Is this correct? If so, how does that work? Are you using some kind of automation based on the awards file?

**Comment by huumn** [comment]
I'm doing it all manually right now which is why the turnaround (eg yesterday's prs) will be variable.

**Comment by felipebueno** [comment]
Cool!

@huumn 

Just clarifying my previous comment. I wasn't pressuring you to pay me or something. I was just curious to know if you were making automatic lightning payments because it's something I'm interested for future projects

**Comment by huumn** [comment]
I can open an issue if you want to write a script that suggests the awards.csv edits as v1 :)

v2 will make automatic payments

**Comment by felipebueno** [comment]
Oh, wow that would be really nice but unfortunately I'll have to deny. I'm not ready to deal with other people's money 

> because it's something I'm interested for future projects

I'm really interested in learning and delving deeper into Lightning Network development but I'll do this slowly and without risking other people's funds

I think I'll wait to read how you (or the devs) would implement it so I can copy it to my little projects 😅

**Comment by huumn** [comment]
@stackernews/foss @stackernews/eng  we have a lot of pending payouts. I've recorded them in `awards.csv` but I'm trying to get a chat system up first. If any of you need payouts sooner than the end of the week, let me know.
